lib/client.js

```diff
@@ -1,22 +1,22 @@
-var version      = require('version'),
-    Promise      = require('../vendor/ayepromise'),
+var _Promise     = require('../vendor/ayepromise'),
```

Comment:
But it worked? Meh, `Promise` has [pretty wide support, except for IE](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise#Browser_compatibility), it's a shame we can't use it and rely on browser availability :crying_cat_face:  Babel can transpile and optionally exclude certain features (if they're already available in target browsers), right?